app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -193,7 +193,21 @@ class GithubWebhookSubscription extends SubscriptionHandler {
   Future<bool> _commitExistsInGob(PullRequest pr) async {
     final RepositorySlug slug = pr.base!.repo!.slug();
     final String sha = pr.mergeCommitSha!;
-    final GerritCommit? gobCommit = await gerritService.findMirroredCommit(slug, sha);
+    GerritCommit? gobCommit;
+    try {
```

Comment:
Should we just allow the exception to propagate? as it is it will fail silently.

Reply:
Not exactly. This exception here is just for the retry. We throw from a call up in the call stack. If the commit is not found then it will return null and we do throw an internal server error here: https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/request_handlers/github/webhook_subscription.dart#L155

Reply:
If the caller is throwing the exception anyway there is no need for the try/catch here. Removing the logic will simplify the code and it will make easier to find potential errors with the real stacktrace in the logs.

Reply:
I see what you mean. I can just change the exception and throw from here.

Reply:
Done. Changed the exception to an internal server error which we throw for simplification.